packages/tasks/src/tasks/audio-classification/data.ts

```diff
@@ -50,16 +54,16 @@ const taskData: TaskDataCustom = {
 	],
 	models: [
 		{
-			description: "An easy-to-use model for Command Recognition.",
+			description: "An easy-to-use model for command recognition.",
 			id: "speechbrain/google_speech_command_xvector",
 		},
 		{
-			description: "An Emotion Recognition model.",
+			description: "An emotion recognition model.",
 			id: "ehcalabres/wav2vec2-lg-xlsr-en-speech-emotion-recognition",
 		},
 		{
 			description: "A language identification model.",
-			id: "facebook/mms-lid-126",
+			id: "speechbrain/lang-id-voxlingua107-ecapa",
```

Comment:
`mms-lid` is better here it's the SoTA and would be better to highlight it here.